.github/workflows/run-ui-test.yml

```diff
@@ -6,6 +6,11 @@ on:
     branches:
       - "develop"
       - "main"
+  schedule:
+    # Run twice a day the sanity check, at 9:13 and 21:13.
+    # You ask why 13? because probably less people schedule their tasks at exactly this time, so I
+    # guess CI is less occupied. And 13 is a lucky number ;)
+    - cron: "13 9,21 * * *"
```

Comment:
question: do we need to refresh the table in looker studio? or why aren't there any results from the weekend?

Reply:
🤔 
<img width="1152" alt="Screenshot 2023-05-29 at 08 50 09" src="https://github.com/appunite/Loudius/assets/18102775/460faed9-00fc-468c-b5e5-0e1ef6e8252f">

Reply:
No, we don't need the dashboard is refreshed every time you open it. There are no results because the PR is not yet merged :)